storybook/.storybook/webpack.config.js

```diff
@@ -0,0 +1,10 @@
+var path = require('path');
```

Comment:
Prefer `const` to `var`:
```suggestion
const path = require('path');
```

nit: no need to require the whole `path` lib if we're only using `resolve`:
```suggestion
const { resolve } = require('path');
```

and then on line 7: `include: resolve('../'),`

Reply:
Will do